.eslintrc.json

```diff
@@ -9,8 +9,7 @@
     "plugin:@typescript-eslint/recommended",
     "plugin:react/recommended",
     "plugin:react-hooks/recommended",
-    "plugin:jsx-a11y/recommended",
-    "plugin:prettier/recommended"
```

Comment:
I personally use it to easily fix with webstorm. other people may too.

please, make this config extended (or conditional with JS) so it will not affect general developer experience